homeassistant/components/frontier_silicon/config_flow.py

```diff
@@ -101,6 +110,43 @@ async def async_step_user(
             step_id="user", data_schema=data_schema, errors=errors
         )
 
+    async def async_step_ssdp(self, discovery_info: ssdp.SsdpServiceInfo) -> FlowResult:
+        """Process entity discovered via SSDP."""
+
+        device_url = discovery_info.ssdp_location
+
+        speaker_name = discovery_info.ssdp_headers.get(SSDP_ATTR_SPEAKER_NAME)
```

Comment:
Let's extract the hostname and abort if its already set up so we don't probe

Examples:
https://github.com/home-assistant/core/blob/1937d803c5f9fba91b4f9c810fa376d1dd8a13ff/homeassistant/components/harmony/config_flow.py#L94
https://github.com/home-assistant/core/blob/1937d803c5f9fba91b4f9c810fa376d1dd8a13ff/homeassistant/components/elkm1/config_flow.py#L169

Reply:
I don't have a `CONF_HOST` to match against. Do you want me to parse it out of the `CONF_WEBFSAPI_URL`?

Reply:
>  Do you want me to parse it out of the `CONF_WEBFSAPI_URL`?

Yes. That was the intent of giving you the elkm1 example.